awx/ui/client/src/notifications/shared/notification-list-init.factory.js

```diff
@@ -49,7 +49,7 @@ export default ['Wait', 'GetBasePath', 'ProcessErrors', 'Rest', 'GetChoices',
             scope.relatednotificationsRemove();
         }
         scope.relatednotificationsRemove = scope.$on('relatednotifications', function () {
-                var columns = ['/notification_templates_success/', '/notification_templates_error/'];
+                var columns = ['/notification_templates_started/', '/notification_templates_success/', '/notification_templates_error'];
```

Comment:
looks like a trailing slash got removed by mistake from the last item in this array